build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -109,14 +107,11 @@ https://github.com/dart-lang/build/blob/master/docs/faq.md#how-can-i-resolve-ski
 
   // Output the digests for transitive modules.
   // These can be consumed for hot reloads.
-  var moduleDigests = <String, String>{};
-  for (var dep in transitiveDeps.followedBy([module])) {
-    var assetId = dep.primarySource.changeExtension(jsModuleExtension);
-    moduleDigests[
-            assetId.path.replaceFirst('lib/', 'packages/${assetId.package}/')] =
-        (await buildStep.digest(assetId)).toString();
-  }
-
+  var moduleDigests = <String, String>{
+    for (var jsId in transitiveJsModules)
+      jsId.path.replaceFirst('lib/', 'packages/${jsId.package}/'):
```

Comment:
The wrapping here makes this a bit weird to follow, consider extracting the key into a function so it doesn't wrap?

Maybe at the same time make this more robust (check the path actually starts with lib before blindly replacing).

Reply:
Refactored to call `ddcModuleName` and extracted a method.